src/sentry/api/serializers/models/rule.py

```diff
@@ -258,7 +258,7 @@ def serialize(self, obj, attrs, user, **kwargs) -> RuleSerializerResponse:
             created_by = None
             if user.id == snooze.get("owner_id"):
                 created_by = "You"
-            else:
+            elif snooze.get("owner_id"):
                 creator = user_service.get_user(snooze.get("owner_id"))
```

Comment:
Can use a walrus here cuz it looks cool 😎
```suggestion
            elif owner_id := snooze.get("owner_id"):
                creator = user_service.get_user(owner_id)
```